respeaker_ros/scripts/respeaker_node.py

```diff
@@ -16,6 +16,7 @@
 import sys
 import time
 from audio_common_msgs.msg import AudioData
+from audio_common_msgs.msg import AudioInfo
```

Comment:
`AudioInfo` is not released on indigo, maybe.
https://github.com/ros-gbp/audio_common-release

Reply:
I see.
`jsk_3rdparty`'s test drop the indigo test.
I added the code to avoid AudioInfo import error.
https://github.com/jsk-ros-pkg/jsk_3rdparty/pull/350/commits/2f87dbbabb22f7d1975bd0d603ff059b2784c77f